docs/release/releases/release-1.12.md

```diff
@@ -0,0 +1,37 @@
+# Cluster API v1.12
+
+## Timeline
+
+The following table shows the preliminary dates for the `v1.12` release cycle.
+
+| **What**                                             | **Who**      | **When**                    | **Week** |
+|------------------------------------------------------|--------------|-----------------------------|----------|
+| Start of Release Cycle                               | Release Lead | Monday 18th August 2025     | week 1   |
+| Schedule finalized                                   | Release Lead | Friday 22nd August 2025     | week 1   |
+| Team finalized                                       | Release Lead | Friday 22nd August 2025     | week 1   |
+| *v1.10.x & v1.11.x released*                         | Release Lead | Tuesday 9th September 2025  | week 4   |
+| *v1.10.x & v1.11.x released*                         | Release Lead | Tuesday 7th October 2025    | week 8   |
+| *v1.10.x & v1.11.x released*                         | Release Lead | Tuesday 4th November 2025   | week 12  |
+| v1.12.0-beta.0 released                              | Release Lead | Tuesday 4th November 2025   | week 12  |
+| Communicate beta to providers                        | Comms Lead   | Tuesday 4th November 2025   | week 12  |
+| Communicate upcoming code freeze to the community    | Comms Lead   | Tuesday 4th November 2025   | week 12  |
+| KubeCon idle week                                    | ---          | ---                         | week 13  |
+| v1.12.0-beta.x released                              | Release Lead | Tuesday 18th November 2025  | week 14  |
+| release-1.12 branch created (**Begin Code Freeze**)  | Release Lead | Tuesday 25th November 2025  | week 15  |
+| v1.12.0-rc.0 released                                | Release Lead | Tuesday 25th November 2025  | week 15  |
+| release-1.12 jobs created                            | Release Lead | Tuesday 25th November 2025  | week 15  |
+| v1.12.0-rc.x released                                | Release Lead | Tuesday 2nd December 2025   | week 16  |
+| **v1.12.0 released**                                 | Release Lead | Tuesday 9th December 2025   | week 17  |
+| *v1.10.x & v1.11.x released*                         | Release Lead | Tuesday 9th December 2025   | week 17  |
+| Organize release retrospective                       | Release Lead | TBC                         | week 17  |
+| *v1.12.1 released (tentative)*                       | Release Lead | Tuesday 16th December 2025  | week 18  |
```

Comment:
nit
```suggestion
| *v1.12.1 released (tentative)*                       | Release Lead | Tuesday 16th December 2025  |       |
```
I would prefer to not give the impression that this week is part of the release cycle, because it is already after GA